install/running_install.adoc

```diff
@@ -655,9 +658,12 @@ intended. However, this is a transient event and the system does correct itself
 over time. See https://github.com/kubernetes/kubernetes/issues/10030 for
 details.
 
-- On failure of the Ansible installer, you must start from a clean operating
-system installation. If you are using virtual machines, start from a fresh
-image. If you are using bare metal machines, see
+- If the Ansible installer fails and you did not need to change a host name or
```

Comment:
"change a host name" is the small part of "sdn configuration", so I think this maybe not accurately.
IMO, It is better to write in this: "If you did not need to change the configuration of SDN"

Reply:
line 663, need to modify in the same way.